protected_environments.go

```diff
@@ -35,8 +35,9 @@ type ProtectedEnvironmentsService struct {
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/protected_environments.html
 type ProtectedEnvironment struct {
-	Name               string                          `json:"name"`
-	DeployAccessLevels []*EnvironmentAccessDescription `json:"deploy_access_levels"`
+	Name                  string                          `json:"name"`
+	RequiredApprovalCount int                             `json:"required_approval_count"`
```

Comment:
This struct here is "only" for representing a protected environment. Can you also change the options for the protect method, here: https://github.com/xanzy/go-gitlab/pull/1482/files#diff-ed40dee592ceb3903d4ec33c1e81474a9362b8dd7f13af1d5f6ca09b2cf200f3R119

Reply:
@timofurrer i have found that other struct and updated it.  Hopefully thats what you meant as your link just seemed to bring me to my original change.

Reply:
In the docs these are ordered the other way around, so `RequiredApprovalCount` comes after `DeployAccessLevels`...